daemons/attrd/attrd_attributes.c

```diff
@@ -312,16 +312,23 @@ attrd_for_cib(const attribute_t *a)
  *                      combination
  */
 static void
-drop_removed_values(const char *cib_id, const char *set_type,
+drop_removed_values(const char *cib_id, const char *set_type, int peer_attrd_ver,
```

Comment:
The new argument needs to go in the Doxygen. The Doxygen also needs to be updated to reflect that we're not deleting only `NULL` attribute values anymore. We'll delete non-`NULL` values if the `peer_attrd_ver` doesn't support clearing transient attributes from the CIB.